### PR TITLE
fix: Tree checkbox animation not correct

### LINKED ISSUE #16259: Tree checkbox animation not correct
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/tree-cn/#components-tree-demo-basic-controlled](https://ant.design/components/tree-cn/#components-tree-demo-basic-controlled)

Kapture 2019-04-23 at 22 23 58

### Steps to reproduce
Click the checkbox twice

### What is expected?
square animation

### What is actually happening?
rect animation

| Environment | Info |
|---|---|
| antd | 3.16.5 |
| React | 16.8.0 |
| System | Mac |
| Browser | Chrome |

### PR DESCRIPTION
**Pull request by DiamondYuan**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Solution

close https://github.com/ant-design/ant-design/issues/16259

### 📝 Changelog

- English Changelog:
- Chinese Changelog (optional):

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by DiamondYuan** [comment]
@zombieJ  https://deploy-preview-16272--ant-design.netlify.com/components/tree-cn/#header

**Comment by zombieJ** [comment]
This failed Checkbox animation:
Kapture 2019-04-24 at 13 51 54

**Comment by zombieJ** [comment]
很多地方用了 `checkbox/style/mixin.less`，看看改哪个最方便些。

**Comment by DiamondYuan** [comment]
> 很多地方用了 `checkbox/style/mixin.less`，看看改哪个最方便些。

全局搜了一下，没注意文件名我就改了。🤦‍♂️

在 tree 里面覆盖了样式。

**Comment by DiamondYuan** [comment]
@zombieJ  可以了

**Comment by zombieJ** [comment]
Tree 还是比正常的大了一圈：
Kapture 2019-04-24 at 16 24 05
Kapture 2019-04-24 at 16 25 06

**Comment by DiamondYuan** [comment]
> Tree 还是比正常的大了一圈：
> Kapture 2019-04-24 at 16 24 05
> Kapture 2019-04-24 at 16 25 06

OK了

File: components/tree/style/index.less [hunk 1]
```diff
@@ -11,6 +11,14 @@
 .antCheckboxFn(@checkbox-prefix-cls: ~'@{ant-prefix}-tree-checkbox');
 
 .@{tree-prefix-cls} {
+  &-checkbox-checked::after {
```
Comment (zombieJ) [inline components/tree/style/index.less]: 这里的数字有点黑科技，注释加一下这个 issue 方便以后有问题排查：
https://github.com/ant-design/ant-design/issues/16259

**Pull request merged**